src/commands/logs.py

```diff
@@ -0,0 +1,363 @@
+import os
+import re
+import sys
+import shutil
+import logging
+import tempfile
+from enum import Enum
+from typing import IO, cast, List, Tuple, TYPE_CHECKING
+from pathlib import Path
+from zipfile import ZipFile, BadZipFile, is_zipfile, ZIP_DEFLATED
+from datetime import date
+
+import aiohttp
+import discord
+from discord import app_commands
+
+from src.constants import TEAM_CHANNEL_PREFIX
+
+if TYPE_CHECKING:
+    from src.bot import BotClient
+
+
+class AnimationHandling(Enum):
+    none = 0
+    team = 1
+    separate = 2
+
+
+logger = logging.getLogger("logs")
+logger.setLevel(logging.INFO)
+handler = logging.StreamHandler(sys.stdout)
+handler.setLevel(logging.INFO)
+logger.addHandler(handler)
+
+# Don't post to team channels and force the guild used so testing can you DMs
+DISCORD_TESTING = bool(os.getenv('DISCORD_TESTING'))
+# Just post all messages to calling channel, allow DMs
+DISCORD_DEBUG = bool(os.getenv('DISCORD_DEBUG'))
+if DISCORD_TESTING or DISCORD_DEBUG:
+    # print all debug messages
+    logger.setLevel(logging.DEBUG)
+    handler.setLevel(logging.DEBUG)
```

Comment:
Thought, would it be worth removing these and making the debug mode settable with an argument on the command